components/badge/badge.css

```diff
@@ -32,12 +32,12 @@
 .badge--status {
   width: 0.8rem;
   height: 0.8rem;
+  border: 1.5px solid var(--bg-color);
 }
 
 .badge--bottom--right {
   bottom: 0.5rem;
```

Comment:
You can change rem to px.

Reply:
Thanks, Saurabh. As of now, I am using px for units less than 5px. Since this is 8px I have used rem.